Sprout/View Controllers/HomeViewController.m

```diff
@@ -0,0 +1,32 @@
+//
+//  HomeViewController.m
+//  Sprout
+//
+//  Created by Sarah Wang on 7/13/21.
+//
+
+#import "HomeViewController.h"
+#import "Parse/Parse.h"
+
+@interface HomeViewController ()
+
+@end
+
+@implementation HomeViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    // Do any additional setup after loading the view.
+}
+
+/*
+#pragma mark - Navigation
+
+// In a storyboard-based application, you will often want to do a little preparation before navigation
+- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
+    // Get the new view controller using [segue destinationViewController].
+    // Pass the selected object to the new view controller.
+}
+*/
```

Comment:
Should be staging line by line to avoid checking in boilerplate code or subclass overrides.